fhir-core/src/main/java/com/ibm/fhir/core/FHIRMediaType.java

```diff
@@ -40,8 +40,7 @@
 
     // Supported values for the MIME-type parameter fhirVersion.
     // https://www.hl7.org/fhir/http.html#version-parameter
-    // The value of this parameter is the publication and major version number for the specification.
     public static final String FHIR_VERSION_PARAMETER = "fhirVersion";
     public static final Set<String> SUPPORTED_FHIR_VERSIONS =
-            Collections.unmodifiableSet(new HashSet<>(Arrays.asList("4.0")));
+            Collections.unmodifiableSet(new HashSet<>(Arrays.asList("4.0","4.0.1")));
```

Comment:
Should it have 4 listed?

Reply:
I haven't found any example of just "4" being a version number, and since the spec, https://www.hl7.org/fhir/http.html#version-parameter, says the value of this parameter is the publication and major version number for the specification, I thought it should just be "4.0", but I since other API operations return "4.0.1" as well, I thought including that would be good to include as well.